Tools/gen_manifest

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import subprocess
+import re
+from shutil import copyfile
+runtime_libs = []
+def init_runtime_libs () :
+  runtime_libs.append('libc')
+  runtime_libs.append('libdl')
+  runtime_libs.append('libm')
+  runtime_libs.append('libpthread')
+  runtime_libs.append('libutil')
+  runtime_libs.append('libnss_dns')
+  runtime_libs.append('libresolv')
+  runtime_libs.append('librt')
+  
+def filter_default_libs (lib_name) :
+  res = True
+  for lib in runtime_libs :
+    if lib_name == lib :
+      res = False
+  return res
+
+def parse_libs (bin_path) :
+  print (bin_path)
+  ldd_out = subprocess.check_output(['ldd', bin_path])
+  lib_list = []
+  for line in ldd_out.splitlines():
+    match = re.match(r'\t(.*) => (.*) \(0x', line)
+    if match and match.group(1) and match.group(2):
+      full_lib_name = match.group(1)
+      name_match = re.match(r'([\w\d]*)(\.*)(.*)', full_lib_name)
+      if name_match:
+        lib_name = name_match.group(1)
+        lib_path = match.group(2)
+        if filter_default_libs (lib_name) : 
+          lib_list.append((name_match.group(1), match.group(2)))
+  return lib_list
+
+def make_exec(path) :
+  mode = os.stat(path).st_mode
+  mode |= (mode & 0o444) >> 2    # copy R bits to X
+  os.chmod(path, mode)
+
+def gen_manifest(app_name, bin_name, g_path) :
+  m_path = g_path + '/graphene/LibOS/shim/test/apps/' + app_name + "/" + app_name +"." + "manifest"
```

Comment:
IMO `g_path` prefix should include `/graphene` part, current version forces user to name Graphene's directory exactly `graphene` (e.g. I have a few different branches checked out to different directories when working on more than one feature simultaneously, all of them named `graphene-something`).

Reply:
Remove the graphene naming restriction